src/libstd/collections/hash/map.rs

```diff
@@ -2462,6 +2463,11 @@ pub struct RandomState {
     k1: u64,
 }
 
+pub(crate) const DETERMINISTIC_HASHING_ENABLED: u8 = 0x01;
```

Comment:
```suggestion
pub(super) const DETERMINISTIC_HASHING_ENABLED: u8 = 0x01;
```

Reply:
Thanks.  I incorporated your suggestion here and at the two other locations where I had used `pub(crate)`.